styles/auth.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .auth-head {
-  margin: 2rem 0;
+  margin: 2rem 0 1rem 0;
 }
```

Comment:
please provide with consistent indentation:)

Reply:
Thanks @j24m  for pointing out this. Will resolve it soon.